README.md

```diff
@@ -9,10 +9,18 @@ The survey microservices for AutoSurvey-968
   - Reactive Web
 * SonarCloud
 * Lombok
+* Swagger
+* DataStax
```

Comment:
Is the datastax driver important enough to list here? Don't really mind either way...

Reply:
I mean that is our link in the code to the DB, I figured the more items in the list the better. Not opposed to trimming it down if anyone else feels the need to take it out